WebApplication/Processing/Arranger.cs

```diff
@@ -134,10 +141,23 @@ public async Task<string> MoveProjectAsync(Project project, string tlaFilename)
 
             var bucket = await _userResolver.GetBucketAsync();
 
+            Func<Task> renameDrawingViewablesAction = async () =>
```

Comment:
I see the approach used twice... maybe it's worth to add optional flag to `RenameObjectAsync()` method?  Like `bool ignoreMissingFile = false`?